samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1964,6 +1965,12 @@ static void run(
       case "transactionLabDoWorkSingleTransactionParallel":
           hubbleTransactionsCodeLab.doWorkSingleTransactionParallel(dbClient, false);
 	break;
+      case "queryLabCreateJoinTables":
+        queryLab.createJoinTables(dbAdminClient, database);
+    break;
+      case "queryLabInsertTestDataToJoinTables":
+        queryLab.insertTestDataToJoinTables(dbClient, 50, 100000);
```

Comment:
Can we name these constants?

Reply:
You mean to create STRING constant variable and then refer them here.
 If that is the ask, then I was thinking constant would help more when we have more occurrence of this value In code, in case of just one occurrence which is here, its fine to keep it like that unless you have some other thoughts?

Reply:
I'm referring to the constants "50" and "1000000". From the callsite, it isn't clear what these constants represent. In fact, I've even forgotten!

There's two ways we generally do this:

1.
```
const Int numMailboxes = 50;
queryLab.insertTestDataToJoinTables(dbClient, numMailboxes, ...);
```

or 2.
```
queryLab.insertTestDataToJoinTables(dbClient, /* numMailboxes */ 50, ...);
```

1 vs. 2  is generally a style choice and will often depend on if we are reusing the constant between many function calls. I'll let you pick.

Reply:
Makes sense I have used both depending upon if it's reusable or not. You might see few extra line of changes (for mutation_per transaction), just don't want to create another PR for it. 
I will take care of other contacts later in one PR